rubin_sim/moving_objects/make_lsst_obs.py

```diff
@@ -226,6 +226,11 @@ def make_lsst_obs():
         dbcols=None,
     )
 
+    # Check if we have "filter" or "band"
+    filter_col = "filter"
+    if filter_col not in pointing_data.dtype.names:
+        filter_col = "band"
```

Comment:
This works, because of the colmap. 
For an old simulation, using the "fbs_sim" colmap ought to be backwards-compatible.